### PR TITLE
Hold status on video call

### PR DESCRIPTION
**Pull request by Semrubo**
VATRP-1767

### CODE REVIEW

File: src/org/linphone/LinphoneManager.java [hunk 1]
```diff
@@ -700,7 +700,7 @@ private void copyAssetsFromPackage() throws IOException {
 		copyIfNotExist(R.raw.linphonerc_default, mLinphoneConfigFile);
 		copyFromPackage(R.raw.linphonerc_factory, new File(mLinphoneFactoryConfigFile).getName());
 		copyIfNotExist(R.raw.lpconfig, mLPConfigXsd);
-		copyIfNotExist(R.raw.rootca, mLinphoneRootCaFile);
+//		copyIfNotExist(R.raw.rootca, mLinphoneRootCaFile);
```
Comment (shareefalis) [inline src/org/linphone/LinphoneManager.java]: revert this change.......

**Comment by kennethdick** [comment]
+1
